core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2451,37 +2454,42 @@ private void loadFileMetadataInternal(RpcContext rpcContext, LockedInodePath ino
     }
 
     // Metadata loaded from UFS has no TTL set.
-    CreateFileOptions createFileOptions = MASTER_OPTIONS.getCreateFileOptions()
-        .setBlockSizeBytes(ufsBlockSizeByte)
-        .setRecursive(options.isCreateAncestors())
-        .setMetadataLoad(true)
-        .setPersisted(true)
-        .setTtl(options.getCommonOptions().getTtl())
-        .setTtlAction(options.getCommonOptions().getTtlAction())
-        .setOwner(ufsStatus.getOwner())
-        .setGroup(ufsStatus.getGroup());
-    short ufsMode = ufsStatus.getMode();
+    CreateFileContext createFileContext = CreateFileContext.defaults();
+    createFileContext.getOptions().setBlockSizeBytes(ufsBlockSizeByte);
```

Comment:
why set createFileContext instead of the way it was?

Reply:
i see. should we set all `options` fields in one go?

Reply:
We didn't have to. However, due to discussions for immutable of options, that's what's going to happen. So yes, we should set it all in one go.